k8s/yaml/servers.yaml

```diff
@@ -1,20 +1,81 @@
 ---
-# Source: seldon-core-v2-servers/templates/seldon-v2-servers.yaml
+# Source: seldon-core-v2-runtime/templates/rclone-gs-public.yaml
```

Comment:
would be better if we separate the runtime from the servers?

Reply:
I wanted to keep the Helm charts to two rather than add a new one but certainly Servers would be added as needed after creating a runtime. Something to discuss and maybe add to a future PR?